inlong-sort/pom.xml

```diff
@@ -35,10 +35,10 @@
         <module>sort-api</module>
         <module>sort-common</module>
         <module>sort-formats</module>
-        <module>sort-connectors</module>
         <module>sort-core</module>
         <module>sort-dist</module>
         <module>sort-end-to-end-tests</module>
+        <module>sort-flink</module>
```

Comment:
please move the `sort-flink` before `sort-dist`.

Reply:
done